journal/modernized/33-chapter-30.adoc

```diff
@@ -746,6 +747,7 @@ and it was perplexing to find that a rough sketch printed by Mariner,
 (who long resided among the Tonga Isles,
 and published a history of them,) in our possession,
 disagreed in toto with two first-rate charts we had on board;
+//toto above is strange, I looked it up and it means "a young one". Doesn't seem to make sense in this sentence but it says this in the pdf too.
```

Comment:
Yeah, I think here in the modernized version let's change `in toto` to `entirely`.

And then, if you don't mind, lets leave the phrase in the original, but wrap it in italics, which is a common convention for denoting little fragments of latin or greek, so it would be:

```ASCIIDOC
disagreed _in toto_ with two first-rate charts we had on board;
```